ui/app/templates/new_user_scheduled_change_modal.html

```diff
@@ -0,0 +1,51 @@
+<div class="modal-header">
+
+    <h3 class="modal-title" ng-show="!sc.username">Add Scheduled Change</h3>
+    <h3 class="modal-title" ng-show="sc.username && !sc.when">Add Scheduled Change for {{ sc.username }}</h3>
+    <h3 class="modal-title" ng-show="sc.username && sc.when">Edit Scheduled Change for {{ sc.username }}</h3>
+</div>
+<div class="modal-body">
+
+  <div ng-show="!loading">
+
+      <div class="tab-content" ng-show="currentItemTab == 1">
+        <form role="form" ng-submit="addScheduledUser()">
+          <div class="form-group" ng-class="{'has-error': errors.username}">
+            <label for="id_username">Username</label>
+            <input type="text" class="form-control" id="id_username" ng-model="permission.username">
+            <p class="help-block" ng-show="errors.username">{{ errors.username.join(', ') }}</p>
+          </div>
+          <div class="form-group" ng-class="{'has-error': errors.permission}">
+            <label for="id_permission">Permission</label>
+            <input type="text" class="form-control" id="id_permission" ng-model="permission.permission">
+            <p class="help-block" ng-show="errors.permission">{{ errors.permission.join(', ') }}</p>
+          </div>
+          <div class="form-group" ng-class="{'has-error': errors.options}">
+            <label for="id_options">Options</label>
+            <textarea ng-model="permission.options" class="options-as-json"></textarea>
+            <p class="help-block" ng-show="errors.options">{{ errors.options.join(', ') }}</p>
+          </div>
+
+        <div class="form-group" ng-class="{'has-error': errors.exception}">
+            <p class="help-block" ng-show="errors.exception">{{ errors.exception }}</p>
+        </div>
+
+          <div style="text-align: right">
+            <input type="hidden" name="data_version" ng-model="permission.data_version">
+          </div>
+        </form>
+
+        </div>
+      </div>
+
+    </div>
+  </div>
+
+  <div loader ng-show="loading"></div>
+
+</div>
+<div class="modal-footer">
+  <div ng-show="saving" small-loader></div>
+  <button type="submit" class="btn btn-primary" ng-click="addScheduledUser()">Save Changes</button>
```

Comment:
I see you're calling this in ng-click and the ng-submit of the form - I think only one of them is necessary - can you double check?